e2e/modeling/test_dpr_training.py

```diff
@@ -0,0 +1,177 @@
+##################################################
```

Comment:
I would remove this test for two reasons. I wonder why anyone would do DPR training in Haystack nowadays. Two years ago - yes. Today I doubt it.  We don't have any asserts here; tests only exercise API. Perhaps not to be too radical, we can leave test_dpr_training as the only "test" and remove everything else.